models/salesforce_history/salesforce__account_daily_history.sql

```diff
@@ -0,0 +1,100 @@
+{{
+    config(
+        enabled = var('salesforce__account_history_enabled', False),
+        materialized = 'incremental',
+        partition_by = {
+            'field': 'date_day', 
+            'data_type': 'date'
+        } if target.type not in ['spark', 'databricks'] else ['date_day'],
+        unique_key = 'account_day_id',
+        incremental_strategy = 'insert_overwrite' if target.type in ('bigquery', 'spark', 'databricks') else 'delete+insert',
+        file_format = 'parquet',
+        on_schema_change = 'fail'
+    )
+}}
+
+with spine as (
+
+    {% if execute %}
+    {% if not var('account_history_start_date', None) or not var('account_history_end_date', None) %}
```

Comment:
Should we also add a config here for the `global_history_start_date`?

Reply:
Additionally, should we include a `global_history_end_date` variable?

Reply:
Config added, but it's not working in the way I'd like.